### PR TITLE
AWS modifications

### LINKED ISSUE #17: [R-T4.2-8] Model API Gateway resources in AWS
| **ID** | **R-T4.2-8** |
|---|---|
| Section | WP4: Modeling Environment |
| Type | FUNCTIONAL_SUITABILITY |
| User Story | As a Software Developer, I want to model API Gateway resources in AWS that trigger functions in AWS Lambda. |
| Requirement | Being able to model API Gateway resources in AWS. |
| Extended Description | A RADON Model should contain an API Gateway resource with respective configuration to trigger a FaaS function hosted in AWS Lambda. |
| Affected Tools | GRAPHMODEL_TOOL |
| Means of Verification | Model review with partners and PRQ case study. |
| Dependency | N/A |

### PR DESCRIPTION
**Pull request by v-yussupov**
This PR introduces several enhancements for AWS-specific node types, namely:

1. AWS API Gateway node type can be used together with AwsApiGatewayTriggers relationship type to generate and deploy APIs for connected AWS Lambda functions.
2. Deployable AWS DynamoDBTable node type is added
3. ServerlessToDoListAPI Service Template is added to demonstrate a simple serverless API deployment using AWS services
4. Ansible implementation of AWS Lambda functions supports attaching artifacts on the level of node templates
5. Other minor modifications in AWS-related types

## Resolving Issue / Feature

As one of the introduced modifications, this PR fixes #17 

- [x] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Ensure to use auto format in **all** files

### CODE REVIEW

**Comment by v-yussupov** [comment]
For compatibility reasons, this PR is to be merged after #37

**Review by miwurster — APPROVED** [review]
Very good job @v-yussupov !!

File: servicetemplates/radon.blueprints/ThumbnailGeneration/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```
Comment (miwurster) [inline servicetemplates/radon.blueprints/ThumbnailGeneration/LICENSE]: Apply the RADON license

File: servicetemplates/radon.blueprints/ServerlessToDoListAPI/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (miwurster) [inline servicetemplates/radon.blueprints/ServerlessToDoListAPI/LICENSE:2]: Apply the RADON license

**Review by miwurster — CHANGES_REQUESTED** [review]

**Review by miwurster — APPROVED** [review]

**Pull request merged**